aws/rust-runtime/aws-sig-auth/src/signer.rs

```diff
@@ -167,7 +167,7 @@ impl SigV4Signer {
             .secret_key(credentials.secret_access_key())
             .region(request_config.region.as_ref())
             .service_name(request_config.service.as_ref())
-            .date_time(request_config.request_ts.into())
+            .time(request_config.request_ts)
```

Comment:
- [ ] just a reminder to me to make sure this is included in the breaking changes list

Reply:
Good catch!